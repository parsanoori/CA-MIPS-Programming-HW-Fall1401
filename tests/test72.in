32
633056730
805578772
443934540
808555946
799682328
220594208
964482181
876171880
788963813
1064812394
103259187
1070144943
1042919944
322587227
520707183
43724547
908144952
607475540
7904767
844629563
685020026
305106184
663868925
263691254
814679942
886689915
4307367
471306708
488477588
933046673
988913576
483755234