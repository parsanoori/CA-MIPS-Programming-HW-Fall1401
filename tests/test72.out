4307367
7904767
43724547
103259187
220594208
263691254
305106184
322587227
443934540
471306708
483755234
488477588
520707183
607475540
633056730
663868925
685020026
788963813
799682328
805578772
808555946
814679942
844629563
876171880
886689915
908144952
933046673
964482181
988913576
1042919944
1064812394
1070144943